### ISSUE TITLE
abnormal adrenal gland zona fasciculata area

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3804]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased adrenal gland zona fasciculata area
2. decreased adrenal gland zona fasciculata area

>2. Synonyms (e.g., absent spleen)

1. enlarged adrenal gland zona fasciculata area
2. reduced adrenal gland zona fasciculata area

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. an increase in size (two-dimensional area) of the wide middle zone of the adrenal cortex that produces cortisol (hydrocortisone)
2. a decrease in size (two-dimensional area) of the wide middle zone of the adrenal cortex that produces cortisol (hydrocortisone)

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

These changes are determined histologically. PMID:36112420 

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal adrenal gland zona fasciculata morphology (MP:0008294)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
http://purl.obolibrary.org/obo/UBERON_0002054

**Comment by sbello** [comment]
@slaulederkind Is there a reason to prefer area over size in the term label? Size is the more common term used in labels. Most terms using area refer to total cell area.

**Comment by sbello** [comment]
looking at annotations also adding
disorganized adrenal gland zona fasciculata

**Comment by sbello** [comment]
Added
abnormal size MP:0014284
increased size MP:0014286
decreased size MP:0014287
disorganized MP:0014285

**Comment by sbello** [comment]
I think size should still work for your annotations. Most of the existing MGI annotations tend to mention increased or expanded or reduced size, so I went with the more general term.
